.scripts/bump_version.py

```
@@ -489,45 +502,58 @@ def bump_version(
-                     "### 🔒 Security",
-                     "### 🚀 Other",
-                 ]
-                 has_auto_content = any(marker in existing_entry for marker in ALL_CATEGORY_MARKERS)
-
-                 if has_auto_content:
-                     print(f"ℹ️ Auto-generated changelog already exists for version {new_version}, skipping auto-generation.")
-                     skip_auto_generation = True
+            # Check if entry for new version already exists and extract it
+            version_header_start = f"## {new_version}"
+            if version_header_start in existing_changelog:
+                print(f"ℹ️ Found existing entry for {new_version}.")
```

Ruff (0.14.10)

String contains ambiguous `ℹ` (INFORMATION SOURCE). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)